wheelsproxy/admin.py

```diff
@@ -160,6 +162,14 @@ def expire_cache_action(self, request, queryset):
     expire_cache_action.short_description = _(
         'Invalidate cache for the selected packages (all platforms)')
 
+    @queryset_action
+    def sync_package_action(self, request, queryset):
+        for package in queryset.iterator():
+            package.index.import_package(p.slug) 
```

Comment:
Better to do this in a task: https://github.com/divio/ac-wheelsproxy/blob/master/wheelsproxy/tasks.py#L35